### ISSUE TITLE
picard locale issue

### ISSUE THREAD

**Issue by alexiswl** [BioContainers/containers#317]
Hi there, I am unable to set the locale in the picard container from quay.

Steps to reproduce:
docker run quay.io/biocontainers/picard:2.18.27--0 picard

/usr/local/bin/picard: line 5: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8): No such file or directory
USAGE: PicardCommandLine <program name> [-h]

I've also seen this converting this image to a singularity container. Since the container has been stripped of tools the issue cannot be resolved by the user.

Alexis

**Comment by osallou** [comment]
Can you open issue on bioconda recipe?

@bgruening this may be a more global issue regarding locale setup on bioconda containers.

**Comment by bgruening** [comment]
Yes, locales are not part of busybox and are not supported. It would make the container significantly bigger than it is now. However, if a tool needs to have it, we can enable an extended-base-image in the meta.yaml of conda that will create a larger image but with locales available.

**Comment by alexiswl** [comment]
Thank you for the quick response, happy to close. 
I was able to create the container through the miniconda3 base image and rebuild the locales from there.  Approx container size was 478 Mb in the end, which isn't too bad.

Alexis.

**Comment by ypriverol** [comment]
@alexiswl would be great if the containers is publish througth conda the corresponding conda recipe as @bgruening suggested.
